spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolume do
+
+  let(:ems_cloud) { FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud) }
+  let(:ems_storage) { FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_storage, :parent_ems_id => ems_cloud.id) }
+  let(:cloud_volume) { FactoryBot.create(:cloud_volume_ibm_cloud_power_virtual_servers, :ext_management_system => ems_storage ) }
+
+  describe "#validate_delete_volume" do
+  context "status is in-use" do
+    let(:status) { "in-use" }
```

Comment:
I don't think this is doing what we want either. I tried using `update_attributes` and it works, e.g. `cloud_volume.update_attributes(:status => "in-use")`.

I went ahead and pushed a working copy of the spec to my fork:
https://github.com/rspradeep/manageiq-providers-ibm_cloud/compare/cloudvolume_spec...jaywcarman:cloudvolume_spec